src/ui_fsmenu/keyboard_options.cc

```diff
@@ -110,7 +160,7 @@ KeyboardOptions::KeyboardOptions(Panel& parent)
                 "keyboard_options",
                 0,
                 0,
-                parent.get_w() * 3 / 4,
+                parent.get_parent()->get_w() * 3 / 4,
```

Comment:
I dont understand. Why only width and not height also? Why in the first place?

Reply:
added a comment to explain